app/models/race_edition.rb

```diff
@@ -9,6 +9,7 @@ class RaceEdition < ActiveRecord::Base
   zonable_attributes :default_start_time_female, :default_start_time_male
 
   accepts_nested_attributes_for :racers
+  accepts_nested_attributes_for :race_entries
```

Comment:
Tell me more about how a model accepts attributes from a list of other models?  I'm confused why the model would accept these (the UI obviously needs fields for multiple models). And why racers is plural?

Actually, why doesn't the UI just collect information for a race entry? That entry references a racer (which is why we have fields to identify the eracer) and a race_edition (which should be automatically identified by the UI). I don't understand why the race_edition would need to be modified.